flocker/ca/functional/test_ca.py

```diff
@@ -100,6 +102,24 @@ def test_control_certificate(self):
         )
 
     @requireCA
+    def test_node_certificate(self):
+        """
+        Test for ``flocker-ca create-node-certificate`` command.
+        Runs ``flocker-ca initialize`` followed by
+        ``flocker-ca create-node-certificate` and calls ``openssl``
+        to verify the generated node certificate and private key is
+        signed by the previously generated certificate authority.
+        """
+        flocker_ca("initialize", "mycluster")
+        status, output = flocker_ca("create-node-certificate")
+        # Find the generated file name with UUID from the output.
+        file_pattern = re.compile("([a-zA-Z0-9\-]*\.crt)")
+        file_name = file_pattern.search(output).groups()[0]
+        self.assertTrue(
+            openssl_verify("cluster.crt", file_name)
+        )
+
```

Comment:
All but one of the tests in this class create the file `flocker/_trial_temp/cluster.crt`. This causes all but the first test executed to give the message ``Error: Unable to write certificate file. File exists``.  Add a cleanup task, either after each initialize line, or separate out the `help_description` test, and add a `tearDown` method.